lsp/core/aws-lsp-json-common/src/language-server/jsonLanguageService.ts

```diff
@@ -0,0 +1,60 @@
+import { AwsLanguageService, SchemaProvider } from '@lsp-placeholder/aws-lsp-core'
+import { JSONDocument, LanguageService, getLanguageService } from 'vscode-json-languageservice'
+import { CompletionList, Diagnostic, FormattingOptions, Hover, Range } from 'vscode-languageserver'
+import { Position, TextDocument, TextEdit } from 'vscode-languageserver-textdocument'
+
+export type JsonLanguageServiceProps = {
+    defaultSchemaUri?: string
+    schemaProvider?: SchemaProvider
+}
+
+export class JsonLanguageService implements AwsLanguageService {
+    private jsonService: LanguageService
+
+    constructor(private readonly props: JsonLanguageServiceProps) {
+        this.jsonService = getLanguageService({
```

Comment:
So this (and the yamlservice) are just wrappers around 3P sevices. That is a key insight that would make the web of classes easier to digest. Could perhaps just be mentioned in the docstring on each class, to start with.

Reply:
Thanks, this is a great call out. I will add the comments